ibis_mssql/tests/test_ops.py

```diff
@@ -0,0 +1,26 @@
+from ibis.tests.expr.mocks import MockConnection
+import unittest
+import pytest
+
+pytest.importorskip('sqlalchemy')
+
+
+class TestOps(unittest.TestCase):
```

Comment:
instead of classes here I think it would be better to just use test function and use conftest for the fixtures.